docs/docs/node-model.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Node Model
+jsdoc: ["gatsby/src/schema/node-model.js"]
+apiCalls: NodeModel
+contentsHeading: Methods
+---
+
+Gatsby exposes its internal data store and query capabilities to GraphQL field resolvers on `context.nodeModel`.
+
+## Example Usage
+
+### gatsby-node.js
+
+```javascript
```

Comment:
Let's use code block title instead of extra h3 heading (which isn't there right now):
````
```javascript:title=gatsby-node.js
````